api/metrics/permissions.py

```diff
@@ -11,6 +11,14 @@ def has_permission(self, request, view):
             return True
         return False
 
+class IsRawMetricsUser(permissions.BasePermission):
```

Comment:
Could we this be more like, IsInstitutionalMetricsUser and use a permission group and not a system tag? You might want to check with @UdayVarkhedkar  to see if this fits the normal user system tag forrmat.

Reply:
I was going with what the Dev Notes in the ticket recommended.

Reply:
A user tag should be fine for this. It's a developer-facing feature rather than a user-facing one, so a quick tag should be okay. The tags that were normalized (that Uday is researching) are for how people get into the OSF, and were normalized because they are used for metrics.